### ISSUE TITLE
Canvas.render() doesn’t allow user to specify outputstream

### ISSUE THREAD

**Issue by davidbalme** [null93/drawille#1]
Currently the render method outputs to System.out..
Would be good if API allows to specify another output stream.. this would allow output to go to a string for example..

**Comment by null93** [comment]
Good idea, I will try to implement it when I get some free time.

**Comment by null93** [comment]
Implemented in v1.0.2 on 033ebca640e4f34a93f613e40f35e6eb93cad145. Canvas.render is now overloaded with _ByteArrayOutputStream_. Example use is as follows:

```java
import io.raffi.drawille.Canvas;

public class Example {
    public static void main ( String [] arguments ) {
        ByteArrayOutputStream stream = new ByteArrayOutputStream ();
        Canvas canvas = new Canvas ( 2, 2 );
        canvas.set ( 1, 1 );
        canvas.render ( stream );
        String result = stream.toString ();
        System.out.println ("Result as string [" + result + "]");
    }
}
```
